src/assets/i18n/fr.json

```diff
@@ -70,37 +70,37 @@
   "Delete user primary group `{name}`": "",
   "Determine how  <a href=\"https://man7.org/linux/man-pages/man1/chmod.1.html\" target=\"_blank\">chmod</a>  behaves when adjusting file ACLs. See the  <a href=\"https://linux.die.net/man/8/zfs\" target=\"_blank\">zfs(8)</a> aclmode property.  <br><br><i>Passthrough</i> only updates ACL entries that are related to the file or directory mode.  <br><br><i>Restricted</i> does not allow chmod to make changes to files or directories with a  non-trivial ACL. An ACL is trivial if it can be fully expressed as a file mode without losing  any access rules. Setting the ACL Mode to Restricted is typically used to optimize a dataset for  SMB sharing, but can require further optimizations. For example, configuring an rsync task with this  dataset could require adding <i>--no-perms</i> in the task <i>Auxiliary Parameters</i> field.": "",
   "Determines the outgoing and incoming traffic ports.<br>  <i>LACP</i> is the recommended protocol if the network switch is capable of  active LACP.<br><i>Failover</i> is the default protocol choice and  should only be used if the network switch does not support active LACP.": "",
-  "Device names of each disk being edited.": "",
+  "Device names of each disk being edited.": "Noms de périphérique de chaque disque en cours d'édition.",
   "Direction": "",
   "Directory Inherit": "",
   "Does your business need <a href=\"https://www.ixsystems.com/truenas/?utm_source=truenas+core+ui&utm_medium=os&utm_campaign=welcome\" target=\"_blank\">Enterprise level</a> support and services? Contact <a href=\"https://www.ixsystems.com/quote-form/?utm_source=truenas+core+ui&utm_medium=os&utm_campaign=welcome\"target=\"_blank\">iXsystems</a> for more information.": "",
   "Domains": "",
   "EC2 Instance ID": "",
   "Edit Portal": "",
   "Either SID or Domain Name + Name are required.": "",
-  "Electing {controller}.": "",
+  "Electing {controller}.": "Élection de {contrôleur}.",
```

Comment:
```diff
- "Electing {controller}.": "Élection de {contrôleur}.",
+ "Electing {controller}.": "Élection de {controller}.",
```
`{controller}` - is a variable and must remain untouched otherwise build would fail

Reply:
Yes sorry, indeed it is a mistake.

Reply:
FIX